src/xls.rs

```diff
@@ -490,7 +490,15 @@ impl<RS: Read + Seek> Xls<RS> {
                                 });
                         formulas.push(Cell::new(fmla_pos, fmla));
                     }
-                    _ => (),
+                    // tests/high_byte_string.xls contains a record type that
+                    // cannot be found in the "By Number" 2.3.2 table
+                    0x00D6 => {
```

Comment:
Tag 0x00D6 is `RSTRING`, i.e., rich multi-format strings.

Reply:
It doesn't seem to be mentioned in the Microsoft online docs (I have an old paper copy of the "Excel Developer's  Kit" and it is in that. Next best to link to would be the OpenOffice docs for the "Microsoft Excel File Format": https://www.openoffice.org/sc/excelfileformat.pdf